obci/interfaces/bci/abstract_classifier.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import abc, copy
+
+
+class AbstractClassifier:
```

Comment:
Klasa AbstractClassifier to interfejs dla algorytmów. Decyzje i ich przesyłanie to klasa AnalysisMaster.